lib/server.js

```diff
@@ -271,6 +271,20 @@ function create(log, error, config, routes, db) {
     }
   )
 
+  const metricsContext = require('./metrics/context')(log, config)
+
+  server.decorate('request', 'stashMetricsContext', request => {
```

Comment:
I like the idea of taking advantage of hapi's ability to add methods to requests. 👍 

I don't like the way this looks, but it's a product of hapi's awkward `decorate` design. Oh well.